### PR TITLE
Added section on the name change in the FAQ

### PR DESCRIPTION
**Pull request by Samuwhale**
Solves #179

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -3,6 +3,13 @@ FAQ
...
 
+**Why did you change the name from MetaSynth to metasyn?**
+
+The project was originally named MetaSynth. However, as we progressed, we discovered that there was already an existing and established audio synthesis software under the same `name <https://uisoftware.com/metasynth/>`_. To avoid potential confusion between these two unrelated projects, we have decided to change the name of our project to **metasyn**. This new name still reflects the package's core goal, of metadata-driven data synthesis. We have also changed the styling of the name to be all lowercase to align with how the package is used in code (e.g. ```import metasyn```).
+
+It is important to note that despite the name change, metasyn as a project and the package's functionality remain the same. We apologize for any confusion caused by the renaming.
```
Comment (qubixes) [inline docs/source/faq.rst:10]: Personally, I don't think we need to apologize!

**Review by qubixes — APPROVED** [review]
Thanks! apart from the comment, it sounds good to me!

File: docs/source/faq.rst (hunk 1 above)
Comment (Samuwhale) [reply docs/source/faq.rst:10]: I agree it's definitely not necessary, but I feel like it is a nice gesture that doesn't hurt. But I can remove it if you'd like?

Comment (qubixes) [reply docs/source/faq.rst:10]: Personally, I feel like it is not necessary with an open source project. "We"'re not really an entity like a corporation would be. Of course for now we only have so many contributors, but maybe in the future there will be many more! (hopefully haha)

Comment (Samuwhale) [reply docs/source/faq.rst]: That makes sense, changed!

**Review by qubixes — APPROVED** [review]
Approved!

**Pull request merged**